flycheck.el

```diff
@@ -8852,7 +8894,7 @@ current versions of MRI and JRuby, but may break when used with
 other implementations or future versions of these
 implementations.
 
-Please consider using `ruby-rubocop' or `ruby-rubylint' instead.
+Please consider using `ruby-rubocop' or `ruby-reek' instead.
```

Comment:
`ruby-lint` is [not actively maintained](https://github.com/YorickPeterse/ruby-lint).